### DISCUSSION TITLE
FIX FOR NO AUDIO  !!!

### DISCUSSION THREAD

**Discussion by Eddieeeeeeeeeeee** [m4dEngi/RemotePlayWhatever#176] [General]
Hello party people,
I found a solution for some games not transmitting the sound.
For me, the problem laid in the "Microsoft Spacial Sound" setting.

To fix it, deactivate spacial sound and you should be good to go.

Once again: Thank you very much for this tool and I really hope that you will keep this updated.

**Comment by Eddieeeeeeeeeeee** [comment]
Addition: I had to deactivate it INSIDE THE GAME!
Not in windows! (Tested it for Uncharted 4 and Ori and the WIll of the Wisps).

**Reply by BIzZmarCk** [reply]
any tipps if spatial sound is disabled in windows and my game does not have such settings?*

**Reply by Squeezitgirdle** [reply]
> any tipps if spatial sound is disabled in windows and my game does not have such settings?*

Enable "play audio on host" under steam settings > remote play settings.
However this will cause an echo as the friend who's remote playing will hear their own voice speaking as an echo. I haven't figured out how to disable that.
